account_invoice_pricelist/__openerp__.py

```diff
@@ -1,38 +1,20 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    Account - Pricelist on Invoices for Odoo
-#    Copyright (C) 2015-Today GRAP (http://www.grap.coop)
-#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
+# -*- coding: utf-8 -*-
+# © 2015-2016 GRAP <http://www.grap.coop>.
+# © 2015-2017 Therp BV <http://therp.nl>.
+# License AGPL-3.0 or later <http://www.gnu.org/licenses/agpl.html>.
 {
     'name': 'Account - Pricelist on Invoices',
-    'version': '8.0.1.0.0',
+    'version': '8.0.1.1.0',
     'summary': 'Add partner pricelist on invoices',
     'category': 'Accounting & Finance',
-    'author': 'GRAP,Odoo Community Association (OCA)',
+    'author': 'GRAP,Therp BV,Odoo Community Association (OCA)',
```

Comment:
OK, no problem. We rework the things here for v9: https://github.com/OCA/account-invoicing/pull/211, and that's why I said that, but without knowing which things you have catched.